libs/base/string_utils.hpp

```diff
@@ -484,101 +444,34 @@ bool ToInteger(char const * start, T & result, int base = 10)
 // negative std::numeric_limits<uint64_t>::max() converts to 1.
 // Values lower than negative std::numeric_limits<uint64_t>::max()
 // are not convertible (method returns false).
-[[nodiscard]] inline bool to_uint64(char const * s, uint64_t & i, int base = 10)
+[[nodiscard]] inline bool to_uint64(std::string_view s, uint64_t & i, int base = 10)
```

Comment:
Ok, and after finalizing the first step, the second commit should remove this function with its comments :)

Reply:
Can do in a follow-up PR.